scheduler/__init__.py

```diff
@@ -0,0 +1,27 @@
+from .scheduler import *
+
+async def replacement_delete_messages(self, messages):
```

Comment:
Holy shit, this scheduler code is a fucking nightmare. And it's awful too apparently, I see it uses that shitty `--start-time` style.

Lets just delete this. It would be saner to copy/paste the v2 bot version and update it to v3, but leave that out of this PR.
https://github.com/tekulvw/Squid-Plugins/blob/master/scheduler/scheduler.py

Reply:
thank god i can't wait to remove that

Reply:
the new one doesn't work but i don't know why.  asynchronous queues/locks aren't something i have any familiarity with

Reply:
the pr can be added without this.  i'll keep working on it, but i don't have an eta on this.  it's unfinished atm.